//basic error operaton
/*
try{
    // throw new Error("its an error");
    // console.log(remon);
}catch(error){
    console.log(error);
    console.log(error.name);
    console.log(error.message);
}
*/

/*
try{
    let age = prompt("enter your age")
    age = Number.parseInt(age);
    if(age>150){
        throw new ReferenceError("Not possible");
    }
}catch(err){
    console.log(err.name);
    console.log(err.message);
    console.log(err.stack);
}

console.log("script is still running");
*/